registration/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
It seems like you haven't written any tests yet. Consider adding test cases by creating classes that inherit from `TestCase` to verify the functionality of your application.